packages/plugins/graphql/package.json

```diff
@@ -48,19 +48,22 @@
     "watch": "strapi plugin:watch"
   },
   "dependencies": {
+    "@apollo/server": "^4.10.0",
```

Comment:
Is the range here on purpose?

Reply:
Same question for every ^ dependency below

Reply:
We could keep a patch range, and freeze both the major and minor versions. What do you think?

Reply:
Nah, I think you've convinced me to drop it altogether 😆

Reply:
graphql I will leave a range because it was before, and since only one specific version may be included in an entire project, we'll leave the flexibility to users to upgrade.